lib/codegen/src/context.rs

```diff
@@ -295,6 +295,7 @@ impl Context {
 
     /// Insert prologue and epilogues after computing the stack frame layout.
     pub fn prologue_epilogue(&mut self, isa: &TargetIsa) -> CtonResult {
+        assert!(self.func.stack_limit.is_none());
```

Comment:
Can you add a brief string here too, like "stack_limit not yet implemented" or so?